lib/rules/attribute-hyphenation.js

```diff
@@ -48,9 +74,14 @@ module.exports = {
     }
 
     function isIgnoredAttribute (value) {
-      if (value.indexOf('data-') !== -1 || value.indexOf('aria-') !== -1) {
+      const isIgnored = !ignoredAttributes.every(function (attr) {
```

Comment:
Perhaps you can use `.some` and remove the negation?
```
ignoredAttributes.some(function (attr) {
  return value.indexOf(attr) !== -1;
});
```

Reply:
That does make more sense, thanks.